### ISSUE TITLE
Not showing Topic after topic_model.get_topic_info()

### ISSUE THREAD

**Issue by hari-chalise** [MaartenGr/BERTopic#2205]
### Discussed in https://github.com/MaartenGr/BERTopic/discussions/2199

<div type='discussions-op-text'>

<sup>Originally posted by **hari-chalise** October 26, 2024</sup>
Hi Maarten, I am trying to Topic modelling BY using BERTopic for Nepali News Article . And now i am just use 1000 Rows of data of news article with two column. But After complete all step embedding, dimensionality reduction, clustering and representation but after running topic_model.get_topic_info() i get nan nan and so on what is my mistake i can't figure out.

![issue_topic_Modelling](https://github.com/user-attachments/assets/d37da367-c093-4987-9bbf-36fc68a43cb0)

 </div>

**Comment by hari-chalise** [comment]
I it required to add whole code here to better clarification?

**Comment by MaartenGr** [comment]
@hari-chalise Yes, please fill in all the information. Which version of BERTopic are you using? What code are you running? Are you running into any errors? If there is an error in the code, I would first need to see it for example.

**Comment by hari-chalise** [comment]
I mean , Can I used BERTopic For Topic Modelling for Nepali News Article ? I was recently tried to do but i encountered error such as Blank in Topic. Currently i am unable to include all error with details code because of my system error so please reply me on my  question as much as possible i will provide details @MaartenGr

**Comment by MaartenGr** [comment]
@hari-chalise 

> I mean , Can I used BERTopic For Topic Modelling for Nepali News Article 

Yes, that should be possible! 

To help you, I will need more information. I need to know how you instantiate the `BERTopic` class. So, what's in the following?

```python
topic_model = BERTopic(
  ...
)
```

I will also need to know which version of BERTopic you are using. 

I can only help you if you provide me that information and then I'll be happy to help you!
